scrapers/dreamsofspanking.yml

```diff
@@ -0,0 +1,32 @@
+name: DreamsOfSpanking
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - dreamsofspanking.com/scene/item
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@id="product"]/h3/text()
+      Date:
+        selector: //div[@class="date"]/text()
+        postProcess:
+          - replace:
+            - regex: .*pdated ([0-9]{1,2} [A-z]{3} [0-9]{4})
+              with: $1
```

Comment:
We want the `created ` date not the updated one. Some scenes dont even have the updated one
The below seems to work ok
```yaml
              - regex: Created\s+(\d+\s[a-zA-Z]{3}\s\d{4}).*
                with: $1
```

Reply:
Some old scenes do miss the updated part, but all the new scenes lack "Created".

Reply:
Are you sure about that ? I checked all the first page and the created date was present.
Did you look at the scene page ? The listing page doesnt matter to us.

Reply:
You're right, I was looking at some single item list pages and didn't see there werent scene pages.
Fixed now